packages/runtime/lifecycle/test/examples/swap.ada.token.spec.e2e.ts

```diff
@@ -44,88 +49,50 @@ describe("swap", () => {
-                (next) => ({
-                  inputs: [pipe(next.applicable_inputs.deposits[0], toInput)],
-                })
-              )
-            ),
-            TE.chainW((contractId) =>
-              runtime(tokenProvider).contracts.applyInputs(contractId)(
-                (next) => ({
-                  inputs: [pipe(next.applicable_inputs.deposits[0], toInput)],
-                })
-              )
-            ),
-            TE.chainW((contractId) =>
-              TE.sequenceArray([
-                pipe(
-                  runtime(adaProvider).payouts.available(
-                    onlyByContractIds([contractId])
-                  ),
-                  TE.map((payoutsAvalaible) => {
-                    expect(payoutsAvalaible.length).toBe(1);
-                    return payoutsAvalaible.map((payout) => payout.payoutId);
-                  }),
-                  TE.chain((payoutIds) =>
-                    runtime(adaProvider).payouts.withdraw(payoutIds)
-                  ),
-                  TE.chain((_) =>
-                    runtime(adaProvider).payouts.withdrawn(
-                      onlyByContractIds([contractId])
-                    )
-                  ),
-                  TE.map((payoutsWthdrawn) =>
-                    expect(payoutsWthdrawn.length).toBe(1)
-                  )
-                ),
-                pipe(
-                  runtime(tokenProvider).payouts.available(
-                    onlyByContractIds([contractId])
-                  ),
-                  TE.map((payoutsAvalaible) => {
-                    expect(payoutsAvalaible.length).toBe(1);
-                    return payoutsAvalaible.map((payout) => payout.payoutId);
-                  }),
-                  TE.chain((payoutIds) =>
-                    runtime(tokenProvider).payouts.withdraw(payoutIds)
-                  ),
-                  TE.chain((_) =>
-                    runtime(adaProvider).payouts.withdrawn(
-                      onlyByContractIds([contractId])
-                    )
-                  ),
-                  TE.map((payoutsWthdrawn) =>
-                    expect(payoutsWthdrawn.length).toBe(1)
-                  )
-                ),
-              ])
-            )
-          )
-      ),
-      TE.match(
-        (e) => {
-          console.dir(e, { depth: null });
-          expect(e).not.toBeDefined();
+      };
+      const swapContract = Examples.SwapADAToken.mkSwapContract(swapRequest);
+      const contractId = await runtime(adaProvider).contracts.create({
+        contract: swapContract,
+        roles: {
+          [swapRequest.provider.roleName]: adaProvider.address,
+          [swapRequest.swapper.roleName]: tokenProvider.address,
         },
-        () => {}
-      )
-    )();
-  }, 1_000_000);
+      });
+      // see [[apply-inputs-next-provider]]
+      await runtime(adaProvider).contracts.applyInputs(contractId, (next) => ({
```

Comment:
can we increase the width for the text formatting ? It doesn't make sense to narrow so much... We have bigger screen and resolution than in 1990 today....

Reply:
reading that is painful ....